### DISCUSSION TITLE
Standalone apps - using a Caddyfile

### DISCUSSION THREAD

**Discussion by bpolaszek** [php/frankenphp#507] [Q&A] [answered]
Sorry, last question for today (I promise!): 😬 

Building standalone apps is cool but is it possible to use a custom Caddyfile (at build time and/or at runtime), especially for custom TLS directives or using specific modules (baldinof/supervisor for instance)?

**Comment by dunglas** [comment, marked as answer]
In the next version, or by using `main`, you will be able to store the custom `Caddyfile` in the root for your embedded and it will work.

Adding custom modules for standalone apps is also possible but a bite more complicated (you'll have to modify the build script by yourself). When the next version of Caddy will be released, we'll be able to use xcaddy in the script, and that should allow to simplify this part.
